orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarter.groovy

```diff
@@ -61,11 +61,19 @@ class DependentPipelineStarter implements ApplicationContextAware {
     pipelineConfig.trigger = new PipelineTrigger(
       parentPipeline,
       parentPipelineStageId,
-      principal?.username ?: user ?: "[anonymous]",
-      buildParameters(pipelineConfig, suppliedParameters),
```

Comment:
This was the problem. It was passing `pipelineConfig` to this method which will read `trigger` from it, but it's doing so while assigning `trigger`. Never going to work.